resources/Khronos-Rejected.css

```diff
@@ -0,0 +1,8 @@
+
+/* Style for a public "Working Draft" */
+
+@import "default.css";
+
+body {
+  background-image: url('logo-Rejected.png');
```

Comment:
Let's just elide the "rejected" logo. It's not worth asking anybody to draw one up. Perhaps consider adding some other banner to make it clear that this extension is not to be implemented.